### PR TITLE
Update Fedora test setup instructions

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -135,9 +135,15 @@ Our puppet modules provide [`Gemfile`](./Gemfile)s which can tell a ruby
...
 
-Please make sure you have [bundler installed](https://bundler.io/#getting-started)
-on your system, then use it to install all dependencies needed for this project,
-by running
+**Prerequisites**
+1. Please make sure you have [bundler installed](https://bundler.io/#getting-started)
+on your system.
+2. If you are using Fedora, you may need these additional packages
+    ```shell
+      % sudo dnf install -y ruby-devel redhat-rpm-config
```
Comment (ekohl) [inline CONTRIBUTING.md]: I typically install `rubygem-bundler` as well instead of installing bundler the regular way.

Comment (sthirugn) [reply CONTRIBUTING.md]: yep, I will add this too!

**Comment by sthirugn** [comment]
@ekohl thanks, updated

**Comment by sthirugn** [comment]
Rebased now, ready to be merged.

**Comment by ehelms** [comment]
@sthirugn can this be closed in favor of a PR to modulesync repository?

**Comment by sthirugn** [comment]
ah missed prior @ekohl comment about this, will work on this now

**Comment by sthirugn** [comment]
Closing this in favor of https://github.com/theforeman/foreman-installer-modulesync/pull/110

**Pull request closed without merging**